src/types/tests/refiner-test.ts

```diff
@@ -492,7 +492,6 @@ describe('refiner', () => {
             particle Foo
                 input: reads Something {num: Number} [ (2+11-9) > num or False ]
         `);
-        viewAst(manifestAst1);
```

Comment:
Sorry! Missed this in my usual grep console to make sure I don't add logs. Thanks for catching it